NitroxClient/Communication/MultiplayerSession/MultiplayerSessionManager.cs

```diff
@@ -12,6 +14,13 @@ namespace NitroxClient.Communication.MultiplayerSession
 {
     public class MultiplayerSessionManager : IMultiplayerSession, IMultiplayerSessionConnectionContext
     {
+        private static readonly Task initSerializerTask;
+
+        static MultiplayerSessionManager()
+        {
+            initSerializerTask = Task.Factory.StartNew(() => Packet.InitSerializer());
```

Comment:
[Better to use `Task.Run` so this task doesn't attach itself to parent tasks (keeping it standalone on the ThreadPool)](https://learn.microsoft.com/en-us/dotnet/standard/parallel-programming/how-to-prevent-a-child-task-from-attaching-to-its-parent)
```suggestion
            initSerializerTask = Task.Run(() => Packet.InitSerializer());
```